fastestimator/op/numpyop/multivariate/read_mat.py

```diff
@@ -28,8 +28,9 @@ class ReadMat(NumpyOp):
     This expects every sample to have a separate .mat file.
 
     Args:
-        file: Dictionary key that contains the .mat path.
-        keys: Key(s) to read from the .mat file.
+        inputs: location of the .mat path.
+        outputs: keys to map the read data to the output.
+        keys: Keys to read from the .mat file.
```

Comment:
Keys should only be required if different from outputs. also, maybe 'mat_keys' to make the distinction more obvious